src/js/components/HostTable.js

```diff
@@ -5,6 +5,7 @@ import {StoreMixin} from 'mesosphere-shared-reactjs';
 import {Tooltip} from 'reactjs-components';
 
 var HostTableHeaderLabels = require('../constants/HostTableHeaderLabels');
+import IconServers from '../../img/components/icons/icon-medium/servers.svg?name=IconServers';
```

Comment:
How does this work, I mean how does it create a component from a plain svg file. I do like it, is this a React functionality?